wishes/views.py

```diff
@@ -0,0 +1,78 @@
+import json, requests
+
+from django.http      import JsonResponse
+from django.views     import View
+from django.db.utils  import IntegrityError
+
+from .models         import Wish
+from products.models import Food
+from users.models    import User
+from users.utils     import login_decorator
+
+class WishView(View):
+    @login_decorator
+    def post(self, request):
+        try:
+            food_id = request.GET['food_id']
+            user    = request.users
+            food    = Food.objects.get(id=food_id)
+
+            if Wish.objects.filter(user=user, food=food_id).exists():
+                return JsonResponse({'message':'FOOD INVALID'}, status=400)
+
+            wish = Wish(
+                user=user,
+                food=food
+            )
+
+            wish.save()
```

Comment:
POST method에서는 food_id를 query parameter로 받아오는게 아니라 payload, 즉 body에 담아서 보내주셔야 해요!

아래와 같이 코드를 수정하면, 코드 수도 줄어들고 더 가독성이 생길 것 같아요! 어떠신가요?

```suggestion
            if Wish.objects.filter(user=request.user, food=food_id).exists():
                return JsonResponse({'message':'DUPLICATED_FOOD'}, status=409)

            wish = Wish(user=user, food=food)
            wish.save()
```

Reply:
더 깔끔해지도록 수정하였습니다. Food_id는 body에서 데이터를 받도록 수정하였습니다. 기존 quary param은 사용하지 않도록 수정하였습니다.